config/ocp/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+bases:
+  - ../../default
```

Comment:
there is an extra `../` in the path. also, bases is deprecated. move directly to be the first value of the `resources`.